### ISSUE TITLE
Multiplayer Scenario Objectives depend on the host

### ISSUE THREAD

**Issue by Konrad22** [wesnoth/wesnoth#3375]
I'm playing Wesnoth on english, so I was very surprised, when my scenario objectives in a multiplayer scenario were written in polish (the host was polish). I don't think that's intended behavior.

**Comment by soliton-** [comment]
This is certainly expected in the general case where your client has just gotten the scenario WML from the host. Fixing this on the network layer is going to be a lot of work and complexity for little gain.

In case the scenario is available locally and has translations it might be reasonable to improve this though.

**Comment by Konrad22** [comment]
In my case I was playing 'Caves of the Basilik', which is why it surprised me even more. ^^

**Comment by Pentarctagon** [comment]
Potentially related to #1420 ?

**Comment by soliton-** [comment]
Yes, a proper fix would require to send untranslated WML and leave translation info in so the receiving client can do the translation.
